web/admin/src/App.js

```diff
@@ -113,85 +118,94 @@ class App extends PureComponent {
             }
           })
           survDraftRef.on('child_removed', data => {
-            this.setState({ surveysDraft: this.state.surveysDraft.filter(x => x.key !== data.key) })
+            this.setState({
+              surveysDraft: this.state.surveysDraft.filter(x => x.key !== data.key),
+            })
           })
         })
       })
     })
   }
 
   render() {
+    const qs = parseQueryString()
     return (
       <div className="App">
-        <Router>
-          <div>
-            <Route
-              exact
-              path="/"
-              render={({ history }) => (
-                <div>
-                  <div className="tableContainer">
-                    <div className="headerRow">
-                      <h2 className="boxTitle">{t('surveys')}</h2>
+        {qs.page ? (
```

Comment:
Let's check that `qs.page` is specifically `exportResultsScreen`.  And since we already know it's a "page", we can call it simply `page=exportResults`

Reply:
Not sure I completely understand the second part